### PR TITLE
DEVELOPER-4053 Added WebpageSearchExclude meta tag

### PR DESCRIPTION
**Pull request by adelasofia**
[DEVELOPER-4053](https://issues.jboss.org/browse/DEVELOPER-4053)

Add `<meta name="DCP:WebpageSearchExclude" content="true">` to webpages that should not appear in the rht_website results. I.e those that have a more specific version of the content.

### CODE REVIEW

**Comment by adelasofia** [comment]
@unibrew Can you take a look to see if this works for you?

**Comment by adelasofia** [comment]
retest this please

File: events/devnation/2016/index.html.slim [hunk 1]
```diff
@@ -6,7 +6,6 @@ interpolate: true
 hero_class: devnation2016
 title: Devnation 2016
 description: Learn more. Code more. Share more. Join the Nation.
-ignore_export: true
```
Comment (LightGuard) [inline events/devnation/2016/index.html.slim]: Why are we removing this?

**Comment by adelasofia** [comment]
@LightGuard You're right, I just added back the `ignore_export: true` to the layouts, but I left the meta tags in case they are ever pushed into drupal again. I'll add the meta tags dirctly to drupal for those pages.

**Comment by LightGuard** [comment]
Looking good.

**Comment by LightGuard** [comment]
retest this please

**Comment by unibrew** [comment]
I took a look at https://developers.redhat.com/video/youtube/rxPctRaTznc/ why this meta tag is somewhere deep inside the <body> instead of <head> ?

**Comment by LightGuard** [comment]
Because adding custom meta tags in Drupal is nontrivial. This was a much easier solution and actually still valid HTML oddly enough.

**Pull request merged**